README.md

```diff
@@ -170,6 +170,13 @@ Values:
 set -g @catppuccin_window_default_text "#{b:pane_current_path}" # use "#W" for application instead of directory
 ```
 
+#### Override the window default separators
+```sh
+set -g @catppuccin_window_default_left_separator ""
+set -g @catppuccin_window_default_middle_separator "█"
+set -g @catppuccin_window_default_right_separator ""
+```
+
```

Comment:
Why not just reuse `catppuccin_window__left/middle/right_separator` and only overwrite for current?

Reply:
It's also possible to do that. `catppuccin_window__left/middle/right_separator` will be taken as default on both options. I think adding the possibility of changing both, give more flexibility.

Reply:
So in this PR `@catppuccin_window_left/middle/right_separator` are only used as default/fallback for the default/current variant?

If that is the case just use just use `@catppuccin_window_left/middle/right_separator` instead of the default variant. Every combination from before remains possible and you have 3 less options that need to be documented and requested from tmux.

Replace the `get_seperator` func with an if on `window_type` to set `separator_type` to `"_current"` and 3 `get_tmux_option` with proper defaults taken from `catppuccin.tmux`.

this way you have 6 options instead of 9 and no hierarchy in the options.

Reply:
Maybe something like this is better:
```sh
if $type == "current"; then
    window_left_separator = get_tmux_option("catppuccin_window_current_left_separator", "$window_left_separator")
    ...
fi

Reply:
I've made an update to match this approach :)